### PR TITLE
v0.15.x rebase

### PR DESCRIPTION
**Pull request by bemyak**
Rebase SharperShape's changes on top of `v0.15.x` branch

### CODE REVIEW

**Review by kszafran — APPROVED** [review]

**Comment by kszafran** [comment]
@bemyak ...Tegola released 0.15.0 the day after you created this PR 😅 https://github.com/go-spatial/tegola/releases/tag/v0.15.0 Maybe you could rebase on top of that?

Also, the reason for this change is that we want to use Postgres 13. I propose to test it as follows:
1. Ask cloudops or Bastien directly to create an `aurora-13` version of our `postgres` image. (BTW, at the same time he can delete the `rds` tag, we don't need it anymore.) Then you can use the rebased tegola in vector-tile-service and run tests. They don't really test much, but will fail if the service fails to connect to the DB.
2. Restore the eu Aurora DB from a snapshot, upgrade it to 13 and temporarily change vector-tile-service env vars in eu to connect to that instance. If you can see Assets in viewer and requests for `.pbf` files in the Chrome Inspector, then the service has successfully connected to Postgres 13.

**Comment by bemyak** [comment]
Git history in the upstream is a bit messed up: the `v0.15.0` tag is assigned to a deattached commit, and `v0.15.x` branch was deleted. I propose keeping this branch, to ease any potential further development.

File: provider/postgis/util.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ import (
 	"strconv"
 	"strings"
 
+	ent "github.com/SharperShape/entity-id"
```
Comment (emreisikligil) [inline provider/postgis/util.go:10]: This is the old library. The new one is `github.com/SharperShape/go-service-common/ent`. We can switch to that one.

Comment (kszafran) [reply provider/postgis/util.go:10]: Nah, let's leave it as is for now.

Comment (kszafran) [reply provider/postgis/util.go:10]: We should not be relying on either here, tbh.

**Comment by emreisikligil** [comment]
> Git history in the upstream is a bit messed up: the v0.15.0 tag is assigned to a deattached commit, and v0.15.x branch was deleted. I propose keeping this branch, to ease any potential further development.

We can also rebase our master onto `v0.15.0` tag since we use master as the main development branch in all other repos. Unless you think it would make things complicated for the next upstream rebase.

File: provider/postgis/util.go (hunk 1 above)
Comment (emreisikligil) [reply provider/postgis/util.go:10]: @kszafran Which repos rely on the old id implementation library? Can we just get rid of it?

Comment (emreisikligil) [reply provider/postgis/util.go:10]: My search shows only this one.

Comment (kszafran) [reply provider/postgis/util.go:10]: Yep, only this one. The `entity-id` repo is already archived.

**Comment by bemyak** [comment]
@emreisikligil, currently our master is in sync with the upstream. The only thing missing from the official `v0.15.0` release is [this minor CI change](https://github.com/go-spatial/tegola/commit/5d447e7e1d8cee3d42b98737844cec5a0fd25278). To me, it seems like the author has pushed the tagged commit and simply forgot to push the master branch. I'd rather wait instead of rebasing.

**Review by emreisikligil — APPROVED** [review]

**Comment by bemyak** [comment]
I've verified that it works with our `vecror-tile-service`, proceeding with merging.

**Pull request merged**